sceptre/scipool/templates/sc-product-s3-private-enc.yaml

```diff
@@ -53,6 +53,10 @@ Resources:
           Info:
             LoadTemplateFromURL: !Sub '${RepoRootURL}/v1.1.10/s3/sc-s3-encrypted-ra.yaml'
           Name: 'v1.1.10'
+        - Description: 'Include option to restrict bucket downloads to same region'
+          Info:
+            LoadTemplateFromURL: !Sub '${RepoRootURL}/1.1.15/s3/sc-s3-encrypted-ra.yaml'
+          Name: 'v1.1.15'
```

Comment:
next version is `v1.1.11`

Reply:
We've gone up to `v1.1.14` according to the tags
https://github.com/Sage-Bionetworks/service-catalog-library/tags

Reply:
ahh ok. lets do `v.1.1.15` then